learn/parse_http.go

```diff
@@ -543,35 +557,40 @@ func parseHeader(header http.Header, responseCode int) []*pb.Data {
 			// Handled by parseBody.
 			continue
 		case "authorization":
-			lv := strings.ToLower(v)
-
-			var authType pb.HTTPAuth_HTTPAuthType
-			var token string
-			if strings.HasPrefix(lv, "bearer ") {
-				authType = pb.HTTPAuth_BEARER
-				token = v[len("bearer "):]
-			} else if strings.HasPrefix(lv, "basic ") {
-				authType = pb.HTTPAuth_BASIC
-				token = v[len("basic "):]
-			} else {
-				authType = pb.HTTPAuth_UNKNOWN
-				token = v
-			}
+			// If the authorization header is in the request, create an
+			// HTTPAuth object.  Treat authorization headers in the response
+			// the same as any other header.
+			if !isResponse {
```

Comment:
```suggestion
			if isRequest {
```

Reply:
Changed in b8861f4f80feecd15ad42c11161843142b3ac2ac.